frontend/src/metabase/parameters/utils/date-formatting.ts

```diff
@@ -3,8 +3,8 @@ import _ from "underscore";
 // eslint-disable-next-line no-restricted-imports -- deprecated usage
 import moment from "moment-timezone";
 
-import { DATE_OPERATORS } from "metabase/query_builder/components/filters/pickers/DatePicker/DatePicker";
-import { EXCLUDE_OPERATORS } from "metabase/query_builder/components/filters/pickers/DatePicker/ExcludeDatePicker";
+import { DATE_OPERATORS } from "metabase/admin/datamodel/components/filters/pickers/DatePicker/DatePicker";
+import { EXCLUDE_OPERATORS } from "metabase/admin/datamodel/components/filters/pickers/DatePicker/ExcludeDatePicker";
```

Comment:
It'd be nice to add named exports here.

```suggestion
import {
  DATE_OPERATORS,
  EXCLUDE_OPERATORS
} from "metabase/admin/datamodel/components/filters/pickers/DatePicker";
```